### DISCUSSION TITLE
In standalone mode, Milvus fails to use GPU.

### DISCUSSION THREAD

**Discussion by aaronyy9** [milvus-io/milvus#31386] [Q&A and General discussion]
1.  Installation reference URL: https://milvus.io/docs/install_standalone-docker-compose-gpu.md
2.  Docker-compose can run, and Milvus can be started, as shown in the screenshot. Compose source address: https://github.com/milvus-io/milvus/releases/download/v2.3.12/milvus-standalone-docker-compose-gpu.yml

However, using nvidia-smi, it is found that Milvus does not utilize the GPU, and no Milvus processes are detected. See screenshot for details.

I seek your assistance, thank you.

![image](https://github.com/milvus-io/milvus/assets/54395520/1b8021ca-f5aa-4b65-9319-15626fa7b753)
![image](https://github.com/milvus-io/milvus/assets/54395520/2ed44de9-afb5-4e15-bdcb-b6afaad23b00)

**Comment by yhmo** [comment]
What kind of index are you using?
Only GPU index can be computed by GPU:
https://milvus.io/docs/index.md#GPU_IVF_FLAT
https://milvus.io/docs/index.md#GPU_IVF_PQ

**Reply by aaronyy9** [reply]
Thank you for your reply. I'd like to confirm: Does this mean that only by using these two types of indexes (GPU_IVF_PQ, GPU_IVF_FLAT) can I utilize GPU resources and see the resource consumption in the nvidia-smi command? Does any other usage of Milvus not occupy GPU resources?

**Reply by yhmo** [reply]
|Index Type| Build Index by Device | Search by Device |
|:-----:|:-----:|:-----:|
| GPU_IVF_FLAT | GPU | GPU |
| GPU_IVF_PQ | GPU | GPU |
| FLAT | CPU | CPU |
| IVF_FLAT | CPU | CPU |
| IVF_SQ8 | CPU | CPU |
| IVF_PQ | CPU | CPU |
| HNSW | CPU | CPU |

**Reply by aaronyy9** [reply]
got it.Thx
